cmd/better-retester/main.go

```diff
@@ -33,6 +36,17 @@ type options struct {
 
 	cacheFile      string
 	cacheRecordAge time.Duration
+
+	enableOnRepo arrayFlags
```

Comment:
You can use Prow's `flagutil.Strings` and then you do not need to reimplement `Set/String` ([example usage](https://github.com/openshift/ci-tools/blob/master/cmd/determinize-ci-operator/main.go#L29))

Reply:
Also, the member is a container, so its name should indicate it (`enableOnRepos`)